cypress/utils/foyer.js

```diff
@@ -61,6 +74,8 @@ const fill_rsa_isolement_recent = (isolement) => {
 export default {
   children,
   kindergartenChildren,
+  deleteChilden,
```

Comment:
il manque un r à remplacer de partout :-)
```suggestion
  deleteChildren,
```